src/hotspot/cpu/aarch64/c1_LIRAssembler_aarch64.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2000, 2018, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2000, 2019, Oracle and/or its affiliates. All rights reserved.
```

Comment:
I don't think we should be updating the copyright dates here.  We've not done that elsewhere to avoid potential merge conflicts (and because we're not Oracle or an affiliate).  Similar comments in other files.